examples/docker/spring/pom.xml

```diff
@@ -51,7 +51,7 @@
             <dependency>
                 <groupId>org.yaml</groupId>
                 <artifactId>snakeyaml</artifactId>
-                <version>1.31</version>
+                <version>1.32</version>
```

Comment:
1.33 is out. In case you want to use that. (just saying, up to you I guess if you want 1.33 or stay with 1.32)

Reply:
Let's update then.